src/Personal/index.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Button } from 'react-native';
+import { useNavigation } from 'react-navigation-hooks';
+import { useSelector } from 'react-redux';
+import RegisterView from './RegisterView';
+import Personal from './personal';
+
+export default function Index() {
+  const { navigate } = useNavigation();
+  const user = useSelector((state) => state.user);
 
-export default function Personal() {
   return (
-    <View>
-      <Text>Personal Screen</Text>
-    </View>
+    (user.accessToken !== '' && user.accessToken !== undefined) ? <Personal /> : (
```

Comment:
please break these lines around the ternary operators, currently it is misleading